csharp/ql/src/change-notes/2024-01-23-threat-models.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Most data flow queries that track flow from *remote* flow sources now use the current *threat model* configuration instead. This doesn't lead to any changes in the produced alerts (as the default configuration is *remote* flow sources) unless the threat model configuration is changed.
```

Comment:
It is worth mentioning the specific queries that also included local sources as default no longer does so under the default threat model configuration (e.g `cs/code-injection`). It is also worth bringing this up in the C# slack channel to make sure that product is informed about this change. IMO this should be perfectly fine as we don't have that many local sources defined in C# as it is (yet) - but we need to make sure that this is properly communicated both to product and via the change note. Also, it seems rather ad hoc that some queries use them while others don't. With the threat models - this will be streamlined.